### DISCUSSION TITLE
The Firacode progress bar is not applied on the terminal.

### DISCUSSION THREAD

**Discussion by frankmdv** [tonsky/FiraCode#1371] [Q&A]
I just installed and applied Firecode Fonts on my Kitty terminal. The fonts were applied but I see that the progress bar of my terminal remains the same as it brings by default, any solution? Thanks in advance.

**Comment by tonsky** [comment]
Fira Code adds new characters to render progress bar with. But existing progress bars will still be displayed the old way, until the programs that produce them upgrade. These are not exactly ligatures, just new separate characters

**Reply by RandomMonkey99** [reply]
Would be great having ligatures
